utils/docker/base/Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN apt-get update \
     cmake \
     vim \
     unzip \
+    basez \
```

Comment:
Added! We should revise our software requirement section and simplify this. As we already recommend Ubuntu as developer OS, we can also add a `apt-get install ... all packages we need` section instead of list there all tiny packages.